package.json

```diff
@@ -28,52 +28,54 @@
   "homepage": "https://github.com/egendata/operator#readme",
   "dependencies": {
     "@egendata/messaging": "0.4.2",
-    "@panva/jose": "1.9.0",
-    "axios": "^0.18.1",
-    "body-parser": "^1.19.0",
+    "@panva/jose": "1.9.2",
+    "axios": "0.19.0",
+    "body-parser": "1.19.0",
     "change-case": "3.1.0",
     "changecase-objects": "1.1.0",
     "cookie-parser": "1.4.4",
     "cors": "2.8.5",
     "dotenv": "8.1.0",
     "dropbox-fs": "0.3.1",
-    "elastic-apm-node": "2.17.0",
+    "elastic-apm-node": "3.0.0",
     "express": "4.17.1",
-    "http-errors": "1.7.2",
+    "http-errors": "1.7.3",
     "memory-fs": "0.4.1",
     "merge": "1.2.1",
     "moment": "2.24.0",
     "morgan": "1.9.1",
-    "node-pg-migrate": "3.20.0",
+    "nconf": "0.10.0",
+    "node-pg-migrate": "3.23.1",
     "pg": "7.12.1",
-    "uuid": "3.3.2"
+    "uuid": "3.3.3",
+    "winston": "3.2.1"
   },
   "devDependencies": {
-    "@commitlint/cli": "^8.1.0",
-    "@commitlint/config-conventional": "^8.1.0",
+    "@commitlint/cli": "^8.2.0",
+    "@commitlint/config-conventional": "^8.2.0",
     "@semantic-release/changelog": "^3.0.4",
-    "@semantic-release/exec": "^3.3.6",
+    "@semantic-release/exec": "^3.3.7",
     "@semantic-release/git": "^7.0.16",
-    "@types/axios": "0.14.0",
-    "@types/ioredis": "4.0.10",
-    "@types/jest": "24.0.12",
-    "@types/supertest": "2.0.7",
-    "eslint": "5.16.0",
-    "eslint-config-standard": "12.0.0",
-    "eslint-plugin-import": "2.18.2",
-    "eslint-plugin-jest": "22.5.1",
-    "eslint-plugin-node": "9.0.1",
-    "eslint-plugin-promise": "4.2.1",
-    "eslint-plugin-standard": "4.0.1",
-    "get-value": "3.0.1",
-    "husky": "^3.0.5",
-    "jest": "24.8.0",
-    "jest-spec-reporter": "1.0.5",
-    "lint-staged": "^9.2.5",
-    "nodemon": "1.19.0",
-    "object-path": "0.11.4",
+    "@types/axios": "^0.14.0",
```

Comment:
were this changes for dev dependencies from being fixed to using `^` intentional ?

Reply:
Yeah... It's my OCD... when stuff is inconsistent I just suffer :(